rspec-buildkite-analytics.gemspec

```diff
@@ -6,12 +6,17 @@ Gem::Specification.new do |spec|
   spec.name          = "rspec-buildkite-analytics"
   spec.version       = RSpec::Buildkite::Analytics::VERSION
   spec.authors       = ["Buildkite"]
-  spec.email         = ["hello@buildkite.com"]
+  spec.email         = ["support+analytics@buildkite.com"]
```

Comment:
Also just saw this and update it so they can be triaged to support quicker if they contact us